pkg/new-ui/v1beta1/backend.go

```diff
@@ -96,6 +111,16 @@ func (k *KatibUIHandler) CreateExperiment(w http.ResponseWriter, r *http.Request
 		return
 	}
 
+	namespace := job.ObjectMeta.Namespace
+	expName := job.ObjectMeta.Name
+
+	err = IsAuthorized(user, "create", namespace, "kubeflow.org", "", "", experimentv1beta1.SchemeGroupVersion, k.sarClient)
```

Comment:
I guess, here you should have `experiments` instead of `kubeflow.org`, right ?

Reply:
Oh, I missed this. Fixed!